test/v1/controllers/ListBsasControllerSpec.scala

```diff
@@ -58,7 +58,7 @@ class ListBsasControllerSpec
 
 
   private val nino = "AA123456A"
-  private val taxYear = "2018-19"
+  private val taxYear = Some("2018-19")
```

Comment:
Earliest tax year for this endpoint is 2019-20 - this should probably be updated to reflect this